.gitignore

```diff
@@ -8,3 +8,5 @@
 .terraform
 *.tfstate
 *.csv
+*.tfstate.*
```

Comment:
Discussed on extra gitignore files and added `*.tfstate.*` and `.terraform.*` to exclude a broader range